package.json

```diff
@@ -97,7 +96,9 @@
     "commit": "git-cz",
     "preci": "rm -fr node_modules",
     "ci": "yarn install --frozen-lockfile",
-    "storybook": "start-storybook -p 6006",
+    "storybook:start": "start-storybook -p 6006",
+    "storybook:test": "npx playwright test --config=storybook-testing/playwright.config.ts",
+    "storybook:test:watch": "yarn chokidar \"src/**/*.{ts*,scss}\" -c 'yarn storybook:test --retries=0 && {path}'",
```

Comment:
i'd watch everything in `src/` not just ts/scss files, you might import a JSON file and then this would not work well

Reply:
or even an svg or another kinds of assets